src/LuaTransformer.ts

```diff
@@ -3515,6 +3515,35 @@ export class LuaTransformer {
                 const firstParamPlusOne = this.expressionPlusOne(params[0]);
                 return this.createStringCall("byte", node, caller, firstParamPlusOne);
             }
+            case "byte":
+            case "char":
+            case "dump":
+            case "find":
+            case "format":
+            case "gmatch":
+            case "gsub":
+            case "len":
+            case "lower":
+            case "match":
+            case "pack":
+            case "packsize":
+            case "rep":
+            case "reverse":
+            case "sub":
+            case "unpack":
+            case "upper":
```

Comment:
Should we allow upper to be redefined? TS already has `toUpperCase()` that should translate to this? Same for several others of these.

Reply:
Personally, I'd like to be able to disable the es libs altogether and just use lua's in some circumstances.